### PR TITLE
fixes and formatting

### PR DESCRIPTION
**Pull request by samwise-nl**
Found a couple issues - main one is that showDiv is duplicated for the journal and core values (gotta be careful of overlapping names - easy to happen when working in separate files). I also did some formatting so it might be hard to see some of the changes but I'll try to point them out.

### CODE REVIEW

File: module1-project.css [hunk 1]
```diff
@@ -232,7 +232,6 @@ body {
 }
 
 #myJournal {
-    display: none;
```
Comment (samwise-nl) [inline module1-project.css:235]: I removed the base display prop and let the function set them. the hidden-div class can be applied to anything you want to hide

File: project1.js [hunk 2]
```diff
@@ -1,133 +1,126 @@
...
 }
 // document.addEventListener("click", displayJournalPrompt)
 
-//Hide/Show Journal Div
-function showDiv() {
-    var div = document.getElementById('myJournal');
-    div.style.display = 'flex';
-    div.scrollIntoView({ behavior: "smooth" });
+//Hide/Show Core Values Div
+function showDivById(divId) {
```
Comment (samwise-nl) [inline project1.js:82]: I renamed this and used it for both the journal and core values - just pass in the id. If you notice you're doing the same thing twice with just a difference of a string/parameter then make it a function and call it.

File: module1-project.html [hunk 3]
```diff
@@ -175,11 +175,7 @@ <h2 id="h2cv">Your Top Six Core Values:</h2>
                 <div class="selected-words"></div>
     
             </div>
-    
-        </div>
         </div>
-    
-        <script src="core-values-java.js"></script>
```
Comment (samwise-nl) [inline module1-project.html:182]: this was causing a duplicate click listener to be added to the core values so it'd add two copies of each word

File: module1-project.css (hunk 1 above)
Comment (colletteap) [reply module1-project.css:235]: Ohh ok, that makes sense. Thanks!

File: project1.js (hunk 2 above)
Comment (colletteap) [reply project1.js:82]: Yes, I knew something was up with duplicating that function. I didn't realize I could do this. Thanks!

File: module1-project.html (hunk 3 above)
Comment (colletteap) [reply module1-project.html:182]: Yes!! I noticed this, as well. Thanks for catching that. I missed deleting that.

**Pull request merged**